openquake/server/db/actions.py

```diff
@@ -237,7 +238,38 @@ def get_outputs(db, job_id):
     return db('SELECT * FROM output WHERE oq_job_id=?x', job_id)
 
 
-DISPLAY_NAME = dict(dmg_by_asset='dmg_by_asset')
+DISPLAY_NAME = {
+    'gmf_data': 'Ground Motion Fields',
+    'dmg_by_asset': 'Average Asset Damages',
+    #'damages_by_asset': 'Average Asset Damages',
+    #'damages_by_event': 'Aggregate Event Damages',
+    'losses_by_asset': 'Average Asset Losses',
+    'losses_by_event': 'Aggregate Event Losses',
+    'damages-rlzs': 'Asset Damage Distribution',
+    'avg_losses-rlzs': 'Average Asset Losses',
+    'avg_losses-stats': 'Average Asset Losses Statistics',
+    'loss_curves': 'Asset Loss Curves',
+    'loss_curves-stats': 'Asset Loss Curves Statistics',
+    'loss_maps-rlzs': 'Asset Loss Maps',
+    'loss_maps-rlzs': 'Asset Loss Maps Statistics',
```

Comment:
Typo: Should be `loss_maps-stats`

Reply:
Fixed in https://github.com/gem/oq-engine/commit/b9d79beae8d86fb95aea312730abc813d2fc31f6